lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb

```diff
@@ -6,7 +6,11 @@ categories: jekyll update
 ---
 You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
 
-To add new posts, simply add a file in the `_posts` directory that follows the convention `YYYY-MM-DD-name-of-post.ext` and includes the necessary front matter. Take a look at the source for this post to get an idea about how it works.
+Jekyll requires blog post files to be named according to the following format:
+
+`YEAR-MONTH-DAY-title.MARKUP`
+
+Where `YEAR` is a four-digit number, `MONTH` and `DAY` are both two-digit numbers, and `MARKUP` is the file extension representing the format used in the file. After that include the necessary front matter. Take a look at the source for this post to get an idea about how it works.
```

Comment:
Being less explicit here is not something I thought would be an issue, but here we are 🤷‍♂️ .

Welcome to the project, @104fps. Only 1 other change and this is 👍🏼.

"After that, include the necessary front matter."